src/shogun2-core/src/main/java/de/terrestris/shogun2/service/PluginService.java

```diff
@@ -95,4 +146,14 @@ public String getPluginSource(String simpleClassName) throws Exception {
 	public void setDao(D dao) {
 		this.dao = dao;
 	}
+
+	/**
+	 * Sets the applicationService.
+	 *
+	 * @param applicationService
+	 */
+	@Autowired
```

Comment:
You MUST declare `@Qualifier('applicationService')` here!
Otherwise spring cannot decide which `ApplicationService` to use (if there is an extension like `ProjectApplicationService`) in your project